Notification.Application/Commands/Broker/NotificationCommand.cs

```diff
@@ -0,0 +1,49 @@
+﻿using MediatR;
+using Notification.Application.Commands.Mail;
+using Notification.Application.Handler;
+using Notification.Application.Logger;
+using Notification.Data.Repositories.Interface;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Notification.Application.Commands.Broker
+{
+    public class NotificationCommand : IRequest
+    {
+        public NotificationCommand(Contracts.DataTransferObject.Broker.Notification freeBook)
+        {
+            FreeBook = freeBook;
+        }
+
+        public Contracts.DataTransferObject.Broker.Notification FreeBook { get; set; }
+    }
+
+    public class NotificationHandler : LoggerRequestHandler<NotificationCommand, Unit>
+    {
```

Comment:
I know that it's I names base class as LoggerBlaBlaBla, but I guess it's better to name something like RequestHandlerBase or something like this. 
Not ASAP, just fyi.